community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tools/gpu-test

```diff
@@ -0,0 +1,76 @@
+# Fail gracefully if nvidia-smi or dcgmi doesn't exist
```

Comment:
Don't we need to add a shebang?

Reply:
That is added in `setup_external.sh` 
https://github.com/mr0re1/hpc-toolkit/blob/d3bd763bfdd68e36c232c4e44d9c7d86f712d6f4/community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/files/setup_external.sh#L29

Reply:
> That is added in setup_external.sh

Frankly I don't see why it matters.

Also, this file needs a license, please add:

```
#!/bin/bash
# Copyright 2024 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```